### ISSUE TITLE
Several Malware Engines Think This is a Trojan

### ISSUE THREAD

**Issue by neostryder** [cnovel/PodcastBulkDownloader#77]
**Describe the bug**
[https://www.virustotal.com/gui/file/ce95fb388cc93e5ae31d3276785f1b8b37720755e6f102bc72b4932d7a2382ab](https://www.virustotal.com/gui/file/ce95fb388cc93e5ae31d3276785f1b8b37720755e6f102bc72b4932d7a2382ab)

**To Reproduce**
1. Scan the file on [VirusTotal.com](https://www.virustotal.com/gui/home/upload)

**Expected behavior**
No malware

**Screenshots**
Just scan it yourself

**System Information:**
Windows 11

**Additional context**
I had an older version installed and it suddenly began showing through AI detection in Malwarebytes, so I checked it on VirusTotal, and there were three engines that detected it.  I then downloaded a fresh version (0.9) and checked with VirusTotal, and now there are 20 engines that detect it as malware.  I know there can be legitimate processes in apps that are detected as malware, but 20 different providers??

**Comment by didyousayboop** [comment]
I have just encountered this issue too. Microsoft Defender on Windows 11 detects PodcastBulkDownloaderCLI.exe as Trojan:Win32/Wacatac.B!ml. It does not detect the non-CLI, GUI PodcastBulkDownloader.exe as malware.

**Comment by cnovel** [comment]
Thanks for the info, this is surely due to pyinstaller (see #38). I'll look into it, but I'm not sure I can find a quick solution...

**Comment by cnovel** [comment]
@neostryder yeah there's no easy fix available, this is due to pyinstaller. I'll have to investigate other methods to convert the .py to .exe, but I don't have time right now. Best thing to do is to ask the AV provider to whitelist the exe files.

**Comment by cnovel** [comment]
@frankyy72 it should download everything. Can you send the rss link so that I can investigate? Thanks.
